### DISCUSSION TITLE
Live connection to Power BI from DAX studio

### DISCUSSION THREAD

**Discussion by albertan352652** [DaxStudio/DaxStudio#1134] [Q&A] [answered]
I need to get data retrieved from Power BI for more records. I usually use Power BI desktop file itself, open it and get it linked to DAX studio. But I was also told I can get DAX Studio connected to Power BI Service or to Live connection. Can somebody explain me in detail how to do that? I'm having trouble locating local host (connecting to Analysis Services) or connecting to Tabular service by putting in powerbi://api.powerbi.com/*** (where to find it?) Thanks

**Comment by marcosqlbi** [comment, marked as answer]
You need a workspace with a Premium capacity or Premium Per User, then use the XMLA endpoint.
More info:
https://daxstudio.org/docs/tutorials/getting-connected/
https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools

**Reply by albertan352652** [reply]
Thank you. Does it also depend on how DAX Studio is installed? (if only as add in in Excel or also available in Power BI Desktop file's view?)

**Reply by AzizQuazi** [reply]
> You need a workspace with a Premium capacity or Premium Per User, then use the XMLA endpoint. More info: https://daxstudio.org/docs/tutorials/getting-connected/ https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools

Is it possible to connect DAX Studio and XMLA endpoints within a Fabric Trial capacity?

**Reply by dgosbell** [reply]
@AzizQuazi - yes Trial Capacities look like any other premium capacity as far as DAX Studio is concerned

**Comment by dgosbell** [comment]
No, the ability to connect to a Power BI XMLA endpoint works for all the different install options. 

The only functionality which is dependent on the way DAX Studio was installed are the ability to:
* connect to Power Pivot models - which require the Excel addin
* integrate into Power BI Desktop external tools - which requires an "All Users" install

see https://daxstudio.org/docs/installation/index/

**Comment by albertan352652** [comment]
Hi Darren, it's been a long time since I contacted you. I wanted to ask you if you have any new or recent videos related to DAX Studio. Can you please post the source where I can refer to documentation and video. Thanks!
